BookShelfBusinessLogic/Book.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
When you sure the task is almost done, I think it's better to erase links on namespaces you don't use.